components/builder-originsrv/src/server/mod.rs

```diff
@@ -310,12 +313,9 @@ impl Dispatcher for OriginSrv {
 
     fn app_init(
         config: Self::Config,
-        router_pipe: Arc<String>,
+        _router_pipe: Arc<String>,
```

Comment:
Why not just remove this arg completely?

Reply:
`app_init` is abstracted from the net crate so you have to accept 2 args, even if you don't use them. The other services still need the router pipe so it can't be refactored completely out.

Reply:
Gotcha, makes sense.